src/main/java/de/danielbechler/diff/node/DiffNode.java

```diff
@@ -444,6 +444,29 @@ public final void visitParents(final Visitor visitor)
 	}
 
 	/**
+	 * If this node represents a bean property this method returns all annotations of its field.
```

Comment:
To avoid confusion, it would be good to mention that this will only work when the field name matches the property name extracted from the getter.